Index: senhas.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/senhas.env b/senhas.env
new file mode 100644
--- /dev/null	(date 1756162810602)
+++ b/senhas.env	(date 1756162810602)
@@ -0,0 +1,3 @@
+MAIL_USERNAME=prmarcelofmoura@gmail.com
+MAIL_PASSWORD=iicefbrsqwjqudvx
+SECRET_KEY=955e2630c4ce08e0b7855eec624f38bc
